### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by taciturnaxolotl** [LunaMultiplayer/LunaMultiplayer#566]
Currently, there hasn't been a commit in 5 months and issues haven't been touched at all. Is this project dead and if so is there a successor?

**Comment by BraveCaperCat2** [comment]
There hasn't been a commit in 8 months now. The project is most likely abandoned.
Even DMP (which apparently has "worse" performance, which it doesn't) has had more recent commits.

**Comment by ItzPray** [comment]
> Currently, there hasn't been a commit in 5 months and issues haven't been touched at all. Is this project dead and if so is there a successor?

Yes, I have made a fork that has plenty of QOL & Stability changes. Contracts fully working and actively fixing bugs.

https://github.com/ItzPray/KSPMulti

**Comment by BraveCaperCat2** [comment]
> > Currently, there hasn't been a commit in 5 months and issues haven't been touched at all. Is this project dead and if so is there a successor?
> 
> Yes, I have made a fork that has plenty of QOL & Stability changes. Contracts fully working and actively fixing bugs.
> 
> https://github.com/ItzPray/KSPMulti

The main LMP repo is actively fixing bugs too, you know

**Comment by ItzPray** [comment]
> > > Currently, there hasn't been a commit in 5 months and issues haven't been touched at all. Is this project dead and if so is there a successor?
> > 
> > Yes, I have made a fork that has plenty of QOL & Stability changes. Contracts fully working and actively fixing bugs.
> > 
> > https://github.com/ItzPray/KSPMulti
> 
> The main LMP repo is actively fixing bugs too, you know

They are fixing issues too slowly, major issues still exist 3 years later. In my fork i have most of the major bugs fixed, with plenty of QOL enhancements and dedicated Mod support for kerbalism & planetary bases.

**Comment by DasSkelett** [comment]
Hi @ItzPray, thanks for stepping in while this project was inactive.
Now with development having picked up again, how about we join forces? Would you be interested in upstreaming your fixes and features?
I think this would benefit us all and the players the most.

If you decide to keep your fork separate, that is of course your right, but in this case I'm asking you to stop using LMP master servers and instead run your own, both to avoid compatibility issues, and because running them costs money, and the people running them pay this out of their own pocket specifically for Luna Multiplayer.

**Comment by ItzPray** [comment]
Hi @DasSkelett, thanks for reaching out. I’d be happy to upstream bug fixes and quality-of-life improvements where they make sense. I’d prefer to keep my repository independent rather than converting it into an LMP fork, but I can start splitting relevant changes into clean PRs against current LMP. For anything that remains intentionally divergent, I’ll keep that on my side, and I’ll disable use of the LMP master servers for my independent fork.

**Comment by gavazquez** [comment]
Hi @ItzPray.

Thanks for improving the multiplayer mods for KSP
I had to stop with the development of LMP some years ago as I had kids and work got in the way.
Besides that, LMP is getting back on track and it recently got an update and fortunately some developers are joining again. 

I'm giving ownership of the repo and push rights to the people who want to contribute and if you want to join us on our discord and do commits to this repo you're more than welcome.

**Comment by ItzPray** [comment]
Hey @gavazquez, really appreciate you reaching out and the context around LMP. Totally understandable stepping back with life getting busy.

That’s great to hear the project is picking up again. I’d definitely be interested in contributing to the repo. Multiplayer in KSP is something I care a lot about, so I’d be glad to help push things forward.

I just joined the Discord as well. Let me know where I can help.
